test/lib/action_cable/subscription_adapter/solid_cable_test.rb

```diff
@@ -136,10 +136,18 @@ class ActionCable::SubscriptionAdapter::SolidCableTest < ActionCable::TestCase
     end
   end
 
+  test "silencing polling queries when there's no Active Record logger" do
```

Comment:
The assertion of this test here isnt super useful as it's just there to appease a minitest warning. Instead we should be asserting that nothing gets raised because thats what we care about. We also shouldnt be diving this deep into private methods. This test should be setup such that we change the logger to be nil, broadcast out some messages, verify the message gets broadcast and assert that nothing gets raised.